tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -1352,6 +1352,36 @@ public function userSharesFileWithUserUsingTheSharingApi(
 		);
 	}
 
+	/**
+	 * @When /^user "([^"]*)" shares the following (?:files|folders|entities) with user "([^"]*)"(?: with permissions (\d+))? using the sharing API$/
+	 * @When /^user "([^"]*)" shares the following (?:files|folders|entities) with user "([^"]*)" with permissions "([^"]*)" using the sharing API$/
+	 *
+	 * @param string $user1
+	 * @param string $user2
+	 * @param TableNode $table
+	 * @param int|string|string[]|int[] $permissions
+	 *
+	 * @return void
+	 */
+	public function userSharesTheFollowingFilesWithUserUsingTheSharingApi(
+		$user1,
+		$user2,
```

Comment:
```suggestion
		$sharer,
		$sharee,
```
having descriptive variable name is recommended